doc/pages/commands.asciidoc

```diff
@@ -142,9 +142,15 @@ of the file onto the filesystem
     set current session name
 
 *kill[!]* [<exit status>]::
+    *alias* quit-all +
```

Comment:
As @alexherbo2 pointed out, there is a subtle difference between `:kill` and closing all clients, so this alias might cause confusion. So the inconsisten naming might be somewhat justified. 

For discoverability, maybe we should cross-reference  `kill` in the `quit*` docstrings... but we don't do that anywhere else so I'm not sure.

> So maybe `quit` (client) could inherit the `kill` (server) action.

I probably misunderstood but do you mean that `quit` should yell if it's closing the last client with a dirty buffer backed by a daemonized server? That would regress 2cd323b314e9f2b2cdf11bbf2974e9390914d5ea.

> I wish there was a simple way to add qa as an alias for quit-all

I personally rarely ever use `:kill`, but I use `:quit` a lot.
So I wouldn't care much for a short convenience alias for `:kill`.